src/ostree.h

```diff
@@ -12,6 +12,7 @@ static const std::string BOOT_BRANCH = "/usr/share/sota/branchname";
 
 struct Ostree {
   static OstreeDeployment *getBootedDeployment(const std::string &path);
+  static OstreeDeployment *getStagedDeployment(const std::string &path, const std::string &ostree_os);
```

Comment:
@cajun-rat A static member in Ostree maybe? Or we can imagine managing multiple OSTree repos?